scripts/ci/gke.sh

```diff
@@ -275,6 +277,53 @@ teardown_gke_cluster() {
     gcloud container clusters delete "$CLUSTER_NAME" --async
 
     info "Cluster deleting asynchronously"
+
+    create_log_explorer_links
+}
+
+create_log_explorer_links() {
+    if [[ -z "${ARTIFACT_DIR:-}" ]]; then
+        info "No place for artifacts, skipping explorer links"
+        return
+    fi
+
+    artifact_file="$ARTIFACT_DIR/gke-logs-summary.html"
+
+    cat > "$artifact_file" <<- HEAD
+<html style="background: #fff">
+    <head>
+        <title><h4>GKE Logs Explorer</h4></title>
+    </head>
+    <body>
+    <ul style="padding-bottom: 28px; padding-left: 30px; font-family: Roboto,Helvetica,Arial,sans-serif;">
+HEAD
+
+    local start_ts
+    start_ts="$(cat /tmp/GKE_CLUSTER_CREATED_TIMESTAMP)"
+    local end_ts
+    end_ts="$(date -u +"%Y-%m-%dT%H:%M:%SZ")"
+    local project
+    project="$(gcloud config get project --quiet)"
+
+    for authUser in {0..2}; do
+        echo \
+\<li\>\
```

Comment:
Why not use a here-doc in this case as well? To avoid having to put everything on a single line?